protos/dcs/mission/v0/mission.proto

```diff
@@ -434,12 +455,47 @@ message StreamUnitsResponse {
 
 message GetScenarioStartTimeRequest {}
 
-message GetScenarioStartTimeResponse {
-  string datetime = 1;
-}
+message GetScenarioStartTimeResponse { string datetime = 1; }
 
 message GetScenarioCurrentTimeRequest {}
 
-message GetScenarioCurrentTimeResponse {
-  string datetime = 1;
+message GetScenarioCurrentTimeResponse { string datetime = 1; }
+
+message AddMissionCommandRequest {
+  string id = 1;
+  string name = 2;
```

Comment:
I would rename these fields to reduce confusion (See the #developers channel on the discord to see what I mean).

id -> "command_name"
name -> "menu_text"

Also. Add documentation to these fields along the lines of :

* https://github.com/DCS-gRPC/rust-server/blob/main/protos/dcs/unit/v0/unit.proto#L70
* https://github.com/DCS-gRPC/rust-server/blob/main/protos/dcs/common/v0/common.proto#L184

Reply:
When I am going to use this API, I'll probably add a JSON encoded payload as the `id`/`command_name` as most of my use cases have both a command name and one or more arguments. So a more general field time like simply `command` might be more appropriate - emphasising that you can add whatever string you like?

Reply:
Another suggestion: ‘key’ for command name? (I would also be encoding a json payload myself)

Reply:
Before we discuss this further. See https://github.com/DCS-gRPC/rust-server/pull/98#issuecomment-1008441891